hack/istio/sail/func-kiali.sh

```diff
@@ -126,11 +126,15 @@ spec:
   external_services:
     grafana:
       enabled: true
+      internal_url: "http://grafana.${CONTROL_PLANE_NAMESPACE}:3000"
+      external_url: "${grafana_external_url}"
       in_cluster_url: "http://grafana.${CONTROL_PLANE_NAMESPACE}:3000"
```

Comment:
I wonder if these shouldn't be also removed?

Reply:
I left those in there on purpose.. because right now this installs Kiali that comes from OperatorHub.io, which today is 1.89 and won't support this new renamed settings. So it still needs in_cluster_url/url in order to work.

When we release the next minor release of Kiali in OperatorHub.io, we can remove in_cluster_url / url here since that new version will support the renamed settings.